src/App.jsx

```diff
@@ -5,6 +5,7 @@ import './App.css';
 import ListedSpeciesPopup from './components/ListedSpecies/ListedSpeciesPopup';
 import Footer from './components/Footer/Footer';
 import Navbar from './components/Navbar/Navbar';
+import NavbarMobile from './components/Navbar/NavbarMobile';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'NavbarMobile' is defined but never used.